vector/src/main/java/im/vector/home/FavoriteRoomFragment.kt

```diff
@@ -0,0 +1,23 @@
+package im.vector.home
+
+import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import im.vector.R
+import kotlinx.android.synthetic.main.fragment_home_individual.*
+import kotlinx.android.synthetic.main.vector_preference_bing_rule.view.*
+
+class FavoriteRoomFragment : BaseNewHomeIndividualFragment() {
```

Comment:
There is repeated code in FavoriteRoomFragment,  InviteRoomFragment and LowPriorityRoomFragment, should this code be centralised into a common super class?